database/sql/V068__executed_protocol_fees.sql

```diff
@@ -0,0 +1,8 @@
+-- Create a new type to store the executed protocol fee and the token the fee is paid in.
+CREATE TYPE FeeAsset AS (
+    amount numeric(78,0),
+    token bytea
+);
+
+-- Executed protocol fees are stored in the same ordering as the protocol fees in the fee_policies database table.
+ALTER TABLE order_execution ADD COLUMN protocol_fees FeeAsset[] NOT NULL DEFAULT '{}';
```

Comment:
Having this default won't be problematic when we retrieve old data? or we know about it and "don't care"? ideally we would be have it as optional first, and then have a cronejob to update it retroactively, but I guess at this point we do not care about previous data, right?

Reply:
The data will be populated from the OnSettlementEventUpdater so populating executed protocol fees for historic data wouldn't be trivial.